source/linux/How_to_Guides/FAQ/How_to_Check_Device_Tree_Info.rst

```diff
@@ -9,81 +9,152 @@ using the entries in /proc/device-tree.
 In the command prompt of Linux, you should see something similar to the
 following in the /proc/device-tree directory:
 
-::
-
-    root@j721e-evm:~# ls /proc/device-tree/
-    #address-cells  firmware                l3-cache0
-    #size-cells     fixedregulator-evm12v0  main_r5fss_cpsw9g_virt_mac0
-    __symbols__     fixedregulator-sd       memory@80000000
-    aliases         fixedregulator-vsys3v3  model
-    chosen          fixedregulator-vsys5v0  name
-    compatible      gpio-keys               pmu
-    connector       interconnect@100000     reserved-memory
-    cpus            interrupt-parent        serial-number
-    dma_buf_phys    ion                     sound@0
-    dummy-panel     l2-cache0               timer-cl0-cpu0
+.. code-block:: console
+
+	root@<machine>:~# ls /proc/device-tree/
+	#address-cells  firmware                l3-cache0
+	#size-cells     fixedregulator-evm12v0  main_r5fss_cpsw9g_virt_mac0
+	__symbols__     fixedregulator-sd       memory@80000000
+	aliases         fixedregulator-vsys3v3  model
+	chosen          fixedregulator-vsys5v0  name
+	compatible      gpio-keys               pmu
+	connector       interconnect@100000     reserved-memory
+	cpus            interrupt-parent        serial-number
+	dma_buf_phys    ion                     sound@0
+	dummy-panel     l2-cache0               timer-cl0-cpu0
 
 Following are some typical usages you might need:
 
-- Let's say you don't know the exact path of the device tree entry you are
-  trying to check. There is a __symbol__ directory in /proc/device-tree, which
-  is very helpful in such cases. It has an entry for each symbol label in the
-  device tree. You can find the exact path for that symbol by running ``cat``
-  command on that entry. Following is an example demonstrating the use:
+Let's say you don't know the exact path of the device tree entry you are
+trying to check. There is a __symbol__ directory in /proc/device-tree, which
```

Comment:
`__symbol__` will be interpreted as an inline bold. Please use the file role to prevent that.

Also, typo. It should be `__symbols__` as displayed in the above code-block

Reply:
oops, cool will fix ^.^